app/src/main/java/de/moyapro/idleworldsim/domain/traits/GrowthModifyingTrait.kt

```diff
@@ -1,11 +1,23 @@
 package de.moyapro.idleworldsim.domain.traits
 
-import kotlin.math.pow
+import de.moyapro.idleworldsim.domain.valueObjects.DeathRate
+import de.moyapro.idleworldsim.domain.valueObjects.GrowthRate
+import de.moyapro.idleworldsim.domain.valueObjects.HungerRate
 
-abstract class GrowthModifyingTrait : Trait() {
-    abstract fun influenceGrowth(growthRate: Double): Double
+open class GrowthModifyingTrait : Trait() {
+    open fun influenceGrowth(growthRate: GrowthRate): GrowthRate = growthRate
+    open fun influenceHunger(hungerRate: HungerRate): HungerRate = hungerRate
+    open fun influenceDying(deathRate: DeathRate): DeathRate = deathRate
 }
 
 object GrowthTrait : GrowthModifyingTrait() {
-    override fun influenceGrowth(growthRate: Double) = growthRate.pow(level + 1)
+    override fun influenceGrowth(growthRate: GrowthRate) = growthRate.pow(level + 1)
+}
+
+object LowDeathRate : GrowthModifyingTrait() {
+    override fun influenceDying(deathRate: DeathRate): DeathRate = deathRate.pow(level * -1 - 1)
```

Comment:
Die Formel pass nicht zu `HighDeathRate`. Soll das so bleiben?

Reply:
Ups. Ich mach noch einen Test dazu.

Reply:
Hmm, hattest du hier noch was angepasst? Sehe aktuell keinen Unterschied.

Reply:
Ich glaube das ist in dem anderen Branch